app/views/foods/index.html.erb

```diff
@@ -0,0 +1,31 @@
+
+<div class='container' >
+    <div class='loginForm'>
+        <h1>Foods Lists</h1>
+        <div class='food-add-button'>
+            <%= button_to link_to 'Add Food', new_user_food_path(@user) %>
+        </div>
+        <table class="foods-table-container">
+            <thead>
+                <tr>
+                    <th>Food</th>
+                    <th>Measurement</th>
+                    <th>Price/Unit</th>
+                    <th>Quantity</th>
+                    <th>Actions</th>
+                </tr>
+            </thead>
+            <tbody>
+            <% @foods.each do |food| %>
+                <tr>
+                    <td><%=food.name %></td>
+                    <td><%=food.measurement_unit%></td>
+                    <td><%=food.price%></td>
+                    <td><%=food.quantity%></td>
+                    <td><%=button_to 'Delete', user_food_path(@user, food), method: :delete, class:'remove-recipe-button'%></td>
```

Comment:
- [ ] I can also delete a food item from the lists created by different users, but owners should only do it.

Reply:
@Deepakdanger, It was a missed, now fixing is done.